sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_qa/_qa.py

```diff
@@ -46,13 +46,15 @@ class QAEvaluator(MultiEvaluatorBase[Union[str, float]]):
     id = "qa"
     """Evaluator identifier, experimental and to be used only with evaluation in cloud."""
 
-    def __init__(self, model_config, **kwargs):
+    def __init__(self, model_config, threshold=3, _higher_is_better=True, **kwargs):
```

Comment:
Can we apply the same pattern of individual threshold here: 

```suggestion
    def __init__(self, model_config, threshold: Optional[dict] = {"groundedness": 3, "relevance": 3, "coherence": 3, "fluency": 3, "similarity": 3, "f1_score": 0.5}, _higher_is_better=True, **kwargs):
```